utils/validate_versions.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Not specific to this change, but some people have been having problems running precommit outside of a venv.